pom.xml

```diff
@@ -12,7 +12,7 @@
     <groupId>com.salesforce.utam</groupId>
     <artifactId>utam-java</artifactId>
     <packaging>pom</packaging>
-    <version>1.1.5-SNAPSHOT</version>
```

Comment:
Revert the changes to the version property here. It needs to be `-SNAPSHOT` until we decide to release.

Reply:
fixed in 02b97836759131ae068d7e2fa86597ab624cdae6